src/main/java/Main.java

```diff
@@ -100,23 +104,54 @@ public static int CrudOption() {
 
     }
 
-    public static void CreatePatient(){
+    public static void createPatient(int ch){
+        String DatabaseName = tables[ch];
+        PatientClass pat = new PatientClass();
+        System.out.println("Enter ID: ");
+        int ID = myInput.nextInt();
```

Comment:
```suggestion
       pat.setID(myInput.nextInt( )); 
```
Instead of putting it in a variable